workers/task-driver/src/tasks/pay_relayer_fee.rs

```diff
@@ -354,7 +364,8 @@ impl PayRelayerFeeTask {
             .clone()
             .ok_or_else(|| PayRelayerFeeTaskError::State(ERR_NO_MERKLE_PROOF.to_string()))?;
 
-        let recipient_decryption_key = self.state.get_fee_decryption_key().await?;
+        let recipient_decryption_key =
+            self.state.get_fee_key().await?.secret_key().expect("decryption key missing");
```

Comment:
nit: can we:
```suggestion
            self.state.get_fee_key().await?.secret_key().ok_or(PayRelayerFeeTaskError::State(ERR_FEE_KEY_MISSING.to_string()))?;
```

I see no need for an `expect` here

Reply:
it's validated above